workflows/files/ZWEWRF03.properties

```diff
@@ -144,6 +144,54 @@ GATEWAY_PORT=7554
 #  Check this option to enable the Single Sign-On.
 APIML_ENABLE_SSO=false
 
+# JOBS_API_PORT
+# Label: Jobs API Port
+# Abstract: Port that the jobs API service will use
+# Category: Explorer variables
+# Description:
+#  Port that the jobs API service will use
+JOBS_API_PORT=8545 # the port the jobs API service will use
+
+#FILES_API_PORT
+# Label: Files API Port
+# Abstract: Port that the files API service will use
+# Category: Explorer variables
+# Description:
+#  Port that the files API service will use
+FILES_API_PORT=8547 # the port the files API service will use
+
+# JES_EXPLORER_UI_PORT
+# Label: JES-explorer UI Port
+# Abstract: Port that the jes-explorer UI service will use
+# Category: Explorer variables
+# Description:
+#  Port that the jes-explorer UI service will use
+JES_EXPLORER_UI_PORT=8546 # the port the jes-explorer UI service will use
+
+#MVS_EXPLORER_UI_PORT
+# Label: MVS-explorer UI Port
+# Abstract: Port that the mvs-explorer UI service will use
+# Category: Explorer variables
+# Description:
+#  Port that the mvs-explorer UI service will use
+MVS_EXPLORER_UI_PORT=8548 # the port the mvs-explorer UI service will use
+
+#USS_EXPLORER_UI_PORT
+# Label: USS-explorer UI Port
+# Abstract: Port that the uss-explorer UI service will use
+# Category: Explorer variables
+# Description:
+#  Port that the uss-explorer UI service will use
+USS_EXPLORER_UI_PORT=8550 # the port the uss-explorer UI service will use
+
+#ZOWE_EXPLORER_FRAME_ANCESTORS
+# Label: Hosts allowed to embed explorers
+# Abstract: List of hosts allowed to embed explorers
+# Category: Explorer variables
+# Description:
+#  Comma seperated list of hosts allowed to embed explorers
+ZOWE_EXPLORER_FRAME_ANCESTORS="${ZOWE_EXPLORER_HOST}:*,${ZOWE_IP_ADDRESS}:*"
```

Comment:
Here you are using variable substitution in the variable input file. However, this substitution is disabled by default. If you want to be able to substitute variables in the VIF file with other variable values, you need to place `IZU_VARIABLE_SUBSTITUTION_ON` somewhere in the file before the substitution is required.

More info here: https://www.ibm.com/support/knowledgecenter/SSLTBW_2.3.0/com.ibm.zos.v2r3.izua700/izuprog_WorkflowsXML_VariablesInputFile.htm

Reply:
I do not plan to replace within a workflow, these are shell variables that will be utilized during instance.env reading. and the plan is the entry stays as is. However thank you for pointing to missing line in the instance.env generation. it is added now.